styx-api-service/src/test/java/com/spotify/styx/api/ApiVersionTestUtils.java

```diff
@@ -29,6 +29,10 @@
  */
 class ApiVersionTestUtils {
 
+  private ApiVersionTestUtils() {
+    throw new UnsupportedOperationException();
```

Comment:
Is this the new best practice? Like making it really impossible to construct an instance. I wish Java had `object`.

Reply:
Yeah, from Effective Java: http://www.informit.com/articles/article.aspx?p=1216151&seqNum=4

Reply:
Ok. I could only recall the private constructor part not the exception. :)